src/core/router/info.h

```diff
@@ -60,8 +60,8 @@ struct RouterInfoTraits
   /// @brief Router Info size constants
   enum Size : std::uint16_t
   {
-    MinBuffer = core::DSA_SIGNATURE_LENGTH,  // TODO(unassigned): see #498
-    MaxBuffer = 2048,  // TODO(anonimal): review if arbitrary
+    MinBuffer = core::EDDSA25519_SIGNATURE_LENGTH,  // TODO(unassigned): see #498
+    MaxBuffer = core::RSASHA5124096_KEY_LENGTH,  // TODO(anonimal): review if arbitrary
```

Comment:
>MaxBuffer = core::RSASHA5124096_KEY_LENGTH

This quickly breaks functionality. See the use-case for MaxBuffer.

Reply:
I didn't realize this breaks functionality. I'll revert MaxBuffer, and do some reading. Is the MinBuffer value still fine?